source/slang/type-layout.h

```diff
@@ -410,6 +424,8 @@ class ProgramLayout : public Layout
     // will (eventually) belong there...
     List<RefPtr<EntryPointLayout>> entryPoints;
 
+    List<RefPtr<GenericParamLayout>> genericEntryPointParams;
```

Comment:
One minor naming thing is that we are calling this "generic entry point params," but they are really "global generic params" because they aren't specific to any one "entry point" (which means "kernel" in the current terminology).